CHANGELOG.md

```diff
@@ -13,12 +13,21 @@ Increment the:
 * MINOR version when you add functionality in a backwards compatible manner, and
 * PATCH version when you make backwards compatible bug fixes.
 
-## [Unreleased]
-
-* [API/SDK] Logger: Support for Instrumentation library ([#1128](https://github.com/open-telemetry/opentelemetry-cpp/pull/1128))
-* [SDK] Add LogLevel to internal_log ([#1147](https://github.com/open-telemetry/opentelemetry-cpp/pull/1147))
-* [API/SDK] Logger: Propagating resources through LoggerProvider ([#1154](https://github.com/open-telemetry/opentelemetry-cpp/pull/1154))
-* [API]: Allow to use external abseil for bazel targets ([#1172](https://github.com/open-telemetry/opentelemetry-cpp/pull/1172))
+## [1.2.0] 2022-01-31
+
+* [CI] Continuous benchmark tests as part of the CI ([#1174](https://github.com/open-telemetry/opentelemetry-cpp/pull/1174))
+* [API] Allow to use external abseil for bazel targets ([#1172](https://github.com/open-telemetry/opentelemetry-cpp/pull/1172))
+* [EXPORTER] Importing gsl::span if std::span is not available ([#1167](https://github.com/open-telemetry/opentelemetry-cpp/pull/1167))
+* [EXPORTER] Synchronized calls to Exporter::Export & Shutdown ([#1164](https://github.com/open-telemetry/opentelemetry-cpp/pull/1164))
+* [EXPORTER] OTLP http exporter block thread  ([#1163](https://github.com/open-telemetry/opentelemetry-cpp/pull/1163))
+* [TESTS] Jaeger: ThriftSender unit test ([#1162](https://github.com/open-telemetry/opentelemetry-cpp/pull/1162))
+* [EXPORTER] InMemorySpanExporter shutdown fix ([#1161](https://github.com/open-telemetry/opentelemetry-cpp/pull/1161))
+* [EXPORTER] Fix leak in Jaeger exporter ([#1160](https://github.com/open-telemetry/opentelemetry-cpp/pull/1160))
+* [TESTS] ZipkinExporter unit-tests ([#1155](https://github.com/open-telemetry/opentelemetry-cpp/pull/1155))
+* [SDK] `BREAKING CHANGE` Logger: propagating resources through LoggerProvider ([#1154](https://github.com/open-telemetry/opentelemetry-cpp/pull/1154))
```

Comment:
This is concerning. I think we need to follow the versioning policy https://github.com/open-telemetry/opentelemetry-cpp/blob/main/Versioning.md#api-and-sdk-compatibility.

Reply:
The breaking changes are for the logs signal which is an [experimental release](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/Versioning.md#release-policy). So users need to manually enable the feature flag (`WITH_LOGS_PREVIEW` macro) to build and use this functionality, and should be aware that the breaking api/abi changes are possible under the feature flag.

Reply:
Then it's not breaking change. Is there a way to avoid calling these out as breaking change for production users?

Reply:
Yes, it doesn't affect API surface of our stable release. It makes sense to put this notice to a separate section for experimental users without distracting most users.